boards/arm/mimxrt1060_evk/pinmux.c

```diff
@@ -219,6 +219,18 @@ static int mimxrt1060_evk_init(const struct device *dev)
 	/* pull up the ENET_INT before RESET. */
 	GPIO_WritePinOutput(GPIO1, 10, 1);
 	GPIO_WritePinOutput(GPIO1, 9, 0);
+
+#if defined(CONFIG_PTP_CLOCK_MCUX)
+	/* GPIO_AD_B1_02 is configured as 1588_EVENT2_OUT */
+	IOMUXC_SetPinMux(IOMUXC_GPIO_AD_B1_02_ENET_1588_EVENT2_OUT, 0U);
+	/* GPIO_AD_B1_02 PAD functional properties */
+	IOMUXC_SetPinConfig(IOMUXC_GPIO_AD_B1_02_ENET_1588_EVENT2_OUT, 0x10B0u);
+	/* GPIO_AD_B1_03 is configured as 1588_EVENT2_IN */
+	IOMUXC_SetPinMux(IOMUXC_GPIO_AD_B1_03_ENET_1588_EVENT2_IN, 0U);
+	/* GPIO_AD_B1_03 PAD functional properties */
+	IOMUXC_SetPinConfig(IOMUXC_GPIO_AD_B1_03_ENET_1588_EVENT2_IN, 0xB0E9);
+#endif
```

Comment:
Please update to the documentation to add these pins

Reply:
@mmahadevan108 , please, expand your comment. I don't understand what you mean.

Reply:
Looks like new pins have been defined here. Please add it to the `index.rst` file in the doc folder

Reply:
OK, I have just force pushed the modified index.rst files. Please, review my attempt and let me know if I should do any additional modifications. If you like what I uploaded just mark the conversation as resolved. Thanks!